CHANGES.md

```diff
@@ -42,7 +42,7 @@
     argument `upper_bound` with the same purpose for newly enabled regression
     scenarios on `ExponentiatedGradient`.
   For a comprehensive overview of available constraints refer to the new [user
-  guide on fairness constraints for reductions methods](https://fairlearn.github.io/user_guide/mitigation.html#reductions).
+  guide on fairness constraints for reductions methods](https://fairlearn.org/user_guide/mitigation.html#reductions).
```

Comment:
This link seems to be broken - I think it should be https://fairlearn.org/main/user_guide/mitigation.html#reductions like in line 57 

@riedgar-ms

Reply:
Well spotted @hildeweerts  :-)  Indeed, that's because of the branch-specific URLs (where `main` reflects latest `main` branch).